common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -1,24 +1,3 @@
-#k_blackrock = {
```

Comment:
It's moved to the left too much.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121759957-4a984680-cb39-11eb-8065-1b86e705c772.png)

</details>